Bloxstrap/App.xaml.cs

```diff
@@ -175,13 +175,8 @@ protected override void OnStartup(StartupEventArgs e)
                 }
                 else
                 {
-                    if (Process.GetProcessesByName(ProjectName).Length > 1 && !LaunchSettings.IsQuiet)
-                        Frontend.ShowMessageBox(
-                            Bloxstrap.Resources.Strings.Menu_AlreadyRunning, 
-                            MessageBoxImage.Information
-                        );
-
-                    Frontend.ShowMenu();
+                    bool showAlreadyRunningWarning = Process.GetProcessesByName(ProjectName).Length > 1 && !LaunchSettings.IsQuiet;

```

Comment:
i need to double check if the quiet launch check actually serves any purpose, since there's no reason at all to launch the menu with the quiet launch flag set

Reply:
the quiet check there should no longer be needed really, we aren't launching any popup anymore